static/app/views/eventsV2/table/columnEditCollection.tsx

```diff
@@ -399,7 +400,8 @@ class ColumnEditCollection extends React.Component<Props, State> {
                 disabled={!canAdd}
                 icon={<IconAdd isCircled size="xs" />}
               >
-                {t('Add an Equation')}
+                {t('Add an Equation')}&nbsp;
+                <StyledFeatureBadge type="beta" />
```

Comment:
Since you're already styling the `FeatureBadge`, you could just add some margin left instead of `&nbsp`.